index.html

```diff
@@ -8,11 +8,82 @@
       <h3 class="c-section-title -light">A nice title condensing what this website is about</h3>
       <h4 class="c-section-subtitle">A very little description and invitation to use the tool to explore countries.</h4>
     </div>
+
+    <div class="c-search -main">
+      <input type="text" name="search" placeholder="Search Country" />
+      <button>Search</button>
+    </div>
   </div>
 </div>
 
 <div class="l-content -main">
   <div class="wrap">
     <h2 class="c-section-title -dark -secondary -highlighted">And index of indexes to understand the world</h2>
+
+    <div class="m-list -home">
+      <div class="list-item">
```

Comment:
With the rscss style, we would name this element only "item".
Then into scss files, we will have m-list > .item
This avoid huge class names and collision between elements into different modules.